From 316d477d0f8dc0936b4e057545ff83f1993047a6 Mon Sep 17 00:00:00 2001
From: Neena Busireddy <neenareddy.busireddy@nxp.com>
Date: Mon, 3 May 2021 11:44:46 -0700
Subject: [PATCH] src/meson.build: Prepend sysroot for the includedir

Need to prepend PKG_CONFIG_SYSROOT_DIR to the dir path for cross compilation

Upstream-Status: Inappropriate [OE-specific]

Signed-off-by: Neena Busireddy <neenareddy.busireddy@nxp.com>
Signed-off-by: Ahsan Hussain <ahsan.hussain@siemens.com>

---
 src/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index 99f64d9..db2f28a 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -1,7 +1,8 @@
 prog_python = find_program('python3')
 
+pkgconf_sysroot = run_command(prog_python, '-c', 'import os; print(os.environ.get("PKG_CONFIG_SYSROOT_DIR"))').stdout().strip()
 vk_xml = join_paths([
-    vulkan_dep.get_pkgconfig_variable('prefix'),
+    pkgconf_sysroot + vulkan_dep.get_pkgconfig_variable('prefix'),
     'share', 'vulkan', 'registry',
     'vk.xml'
     ])
